packs/feat-effects/effect-heavens-thunder.json

```diff
@@ -50,6 +50,58 @@
                 "selector": "strike-damage",
                 "slug": "heavens-thunder-sonic",
                 "value": "@weapon.system.damage.dice"
+            },
+            {
+                "alwaysActive": true,
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Heaven's Thunder",
+                "option": "heavens-thunder",
+                "placement": "spellcasting",
+                "suboptions": [
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.TraitSonic",
+                        "value": "sonic"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    "item:slug:qi-blast"
+                ],
+                "property": "damage-type",
+                "selectors": [
+                    "spell-damage",
+                    "inline-damage"
```

Comment:
Is inline damage here a mistake or is there an inline damage roll this would be hitting?